chore(.storybook): forenkle og standardiser live code editorer

hagenek opened the pull request:
**Nye live code editorer opprettet for 12 pakker:**

1. ffe-accordion-react: Enkelt, multiple og interaktive eksempler
2. ffe-account-selector-react: Komprehensiv editor for alle selector-komponenter
3. ffe-buttons-react: Alle button-typer med interaktive eksempler
4. ffe-cards-react: Card-varianter og layouts
5. ffe-chips-react: Enkelt template med grunnleggende chip-eksempler
6. ffe-form-react: Komprehensiv form-komponenter med validering
7. ffe-grid-react: Layout og responsive eksempler
8. ffe-icons-react: Icon-bruk eksempler
9. ffe-lists-react: Liste-komponenter
10. ffe-messages-react: SystemMessage og ContextMessage templates
11. ffe-tabs-react: Enkelt template med tab-navigasjon
12. ffe-tags-react: Enkelt template med tag-visning

**Infrastruktur og verktøy:**

- Opprettet LiveCodeEditor.tsx utility for konsistent implementasjon
- Lagt til dokumentasjon i 
- LIVE_CODE_GUIDE.md
- Audit-script (audit-live-code.js) for å spore implementering
- Genererings-script (generate-livecode.js) for automatisering
- Konsistent filnavn: LiveCodeEditor.stories.tsx for alle pakker

**Designprinsipper:**

1. Fokus på 'enkelt' template som standard for enkel bruk
2. Monaco editor integrering for syntax highlighting
3. Real-time preview funksjonalitet
4. Konsistent norsk navngiving og struktur
5. Progressive kompleksitet fra enkle til avanserte eksempler

**Resultat:** 

Introduserer en standardisert live code editor løsning som dekker 12 pakker og forbedrer utvikleropplevelsen betydelig på tvers av hele FFE design systemet.

hagenek committed:
chore(.storybook): forenkle og standardiser live code editorer

tuva-odegard commented:
Såå nice at du har fått det opp! 
Klikket meg litt rundt, det dukket opp noen ting: 
1. Jeg prøvde meg på accordion, det interaktive eksempelet fungerer ikke 
<img width="1367" height="432" alt="image" src="https://github.com/user-attachments/assets/8c692a74-7635-4bd0-ba26-77d19947631a" />

2. Ikonet er ikke der på knappene
<img width="320" height="84" alt="image" src="https://github.com/user-attachments/assets/89b5269e-b2b7-4f11-8c64-97a8fec1c1b1" />

3. Når jeg gjør en endring står det at jeg har ulagrede endringer, men at koden lagres automatisk 🤔 
<img width="626" height="212" alt="image" src="https://github.com/user-attachments/assets/9e1d1a98-037a-4cb7-9032-787741d870b6" />

Skjønner ikke hva jeg skal gjøre som bruker da..

4. Trenger vi "last ned" knappen? De fleste kopierer vel bare? 🤔 
5. Det her burde vel være en ffe-dropdown med tydelig label? 
<img width="185" height="67" alt="image" src="https://github.com/user-attachments/assets/ce1221a9-e609-432d-9905-aceccdddbb99" />

6. Button-eksempelet bruker ikoner vi ikke støtter 🤔 
<img width="561" height="215" alt="image" src="https://github.com/user-attachments/assets/2e81f5f8-e0e9-4689-a69f-c7b728c6a087" />

Burde vel kanskje bruke kun ting fra designsystemet ❤️ 
Har ikke sett igjennom resten men det er kanskje en start? 🙌

hagenek committed:
feat: forbedre LiveCodeEditor med bekreftelsesdialog og tastaturstøtte

- Fjern icon-egenskaper fra alle LiveCodeEditor-maler
- Fjern emoji-ikoner fra template-kode i grid-react og andre pakker
- Erstatt native confirm() med tilpasset bekreftelsesdialog for tilbakestill-knappen
- Legg til Enter/Escape tastaturstøtte for bekreftelsesdialogene
- Gjør Template.icon valgfri i shared LiveCodeEditor interface

tuva-odegard commented:
Flott med endringer! Det er en veldig stor endring, det er kult! Noen ting til 😄 (nummererer bare så det er lettere å skille de fra hverandre)

1. Ser at det ikke fungerer så bra med accent mode, kanskje man ikke trenger live code i alle modes og kontekster, altså at det ikke trenger å være en story? 
<img width="812" height="463" alt="image" src="https://github.com/user-attachments/assets/fc3446a2-3609-4041-be3f-37587c119280" />

2. Ser ikke ut som at Lagre fungerer helt som tiltenkt 🤔 
<img width="1053" height="473" alt="image" src="https://github.com/user-attachments/assets/cb6aa904-fb99-4dba-8da0-8af3740f5ddd" />
Står "Lagret" men fikk opp denne beskjeden ❤️ 
Auto-save ser ikke ut som at den gjør noe 🤔 


3. Ser også ut som at Accordion har byttet på "grunnleggende" og "avansert" eksempelet. Det er vel egentlig ikke vits med en gunnleggende, de var veldig like 🤔 
<img width="1255" height="469" alt="image" src="https://github.com/user-attachments/assets/5c4ac75d-f621-4135-b75a-99cee3aecb10" />

4. Hadde det vært mulig å bruke våre egne farger på cardene? 😄 
<img width="1217" height="359" alt="image" src="https://github.com/user-attachments/assets/602c3203-11d2-42c5-b11e-4165235ccca1" />

5. Kan vi ha "preview" på norsk? 
<img width="614" height="159" alt="image" src="https://github.com/user-attachments/assets/c2638b2b-ef96-4a09-a5d5-58181ddaf9f3" />
😄 

6. Flere av accountselectoreksempelet er brukket :( 
<img width="1229" height="588" alt="image" src="https://github.com/user-attachments/assets/77ed7712-e768-4d20-b6cd-535440196b90" />
Og ett av de ser ikke så bra ut.. 
<img width="1180" height="592" alt="image" src="https://github.com/user-attachments/assets/b1602646-aaac-4543-a291-bf1f7be9ddfd" />

7. F.eks Cards: Trenger kanskje ikke dropdownen når det bare er ett eksempel? 🤔 
<img width="354" height="127" alt="image" src="https://github.com/user-attachments/assets/34c5d40c-af07-4632-be14-3f442840db92" />

Ser ut som det fortsatt er mange av eksemplene som bruker ikoner vi ikke støtter ❤️ Men vet ikke om du var ferdig da!

hagenek committed:
refactor(.storybook): fjern emojier og forbedre UX for LiveCodeEditor

- Fjern alle emojier fra LiveCodeEditor.stories.tsx filer
- Skjul template-dropdown når kun én template eksisterer
- Flytt Tilbakestill-knapp til øvre høyre hjørne av kodeeditoren
- Forbedre dialog-tekster og oversettelser til norsk
- Standardiser import-rekkefølge og code formatting
- Rette template-referanse feil i Accordion stories
- Forenkle AccountSelector templates ved å fjerne komplekse eksempler

Endringene gjør grensesnittet renere og mer profesjonelt ved å fjerne
dekorative emojier, samtidig som UX forbedres med betinget visning av
kontroller og bedre posisjonering av handlingsknapper.

hagenek commented:
> Flott med endringer! Det er en veldig stor endring, det er kult! Noen ting til 😄 (nummererer bare så det er lettere å skille de fra hverandre)
> 
> 1. Ser at det ikke fungerer så bra med accent mode, kanskje man ikke trenger live code i alle modes og kontekster, altså at det ikke trenger å være en story?
> 
> <img alt="image" width="812" height="463" src="https://private-user-images.githubusercontent.com/155720728/483514425-fc3446a2-3609-4041-be3f-37587c119280.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzcsIm5iZiI6MTc1NjQ2MzIzNywicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE0NDI1LWZjMzQ0NmEyLTM2MDktNDA0MS1iZTNmLTM3NTg3YzExOTI4MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYjYwYmVmMTIxZTI5MDNhMDI2ZTAwZGE4MDE1YzY0ZDFhNzgxNTNhYTI4OWUzMWE5Y2IwYWZmMDJmZjdiYmQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LzbLL3nTudI5Am9nUeTvPhkahe1cVEVgkqAgSJRtseE">
> 2. Ser ikke ut som at Lagre fungerer helt som tiltenkt 🤔
> 
> <img alt="image" width="1053" height="473" src="https://private-user-images.githubusercontent.com/155720728/483514643-cb6aa904-fb99-4dba-8da0-8af3740f5ddd.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzcsIm5iZiI6MTc1NjQ2MzIzNywicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE0NjQzLWNiNmFhOTA0LWZiOTktNGRiYS04ZGEwLThhZjM3NDBmNWRkZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNTFhYzQ1ZTU4N2NmM2FjNTA0MmYzZTA4MjJmYjAzOGE0ZWJmMTk1N2Q4YmY0ODYzN2VlZjFhZDUzZDE1MDA2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.4R6anndt3YeOZoPK9Avj_pGS8tCaGiXJyWadrvCLaJ4"> Står "Lagret" men fikk opp denne beskjeden ❤️ Auto-save ser ikke ut som at den gjør noe 🤔
> 3. Ser også ut som at Accordion har byttet på "grunnleggende" og "avansert" eksempelet. Det er vel egentlig ikke vits med en gunnleggende, de var veldig like 🤔
> 
> <img alt="image" width="1255" height="469" src="https://private-user-images.githubusercontent.com/155720728/483515088-5c4ac75d-f621-4135-b75a-99cee3aecb10.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzcsIm5iZiI6MTc1NjQ2MzIzNywicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE1MDg4LTVjNGFjNzVkLWY2MjEtNDEzNS1iNzVhLTk5Y2VlM2FlY2IxMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYjMzN2FhNjliYmE2Mzg0ZGVmMzIyZTY1OTU1ZTY4Y2QzNDgyMGFkMWUwOGZhYjdjMDU5ZmUzNzUyMmUyZmVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.N-1eS8qaSQC0h0Vrlqea0Ifq5cIfBDsmdn4elt-HszM">
> 4. Hadde det vært mulig å bruke våre egne farger på cardene? 😄
> 
> <img alt="image" width="1217" height="359" src="https://private-user-images.githubusercontent.com/155720728/483515507-602c3203-11d2-42c5-b11e-4165235ccca1.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzcsIm5iZiI6MTc1NjQ2MzIzNywicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE1NTA3LTYwMmMzMjAzLTExZDItNDJjNS1iMTFlLTQxNjUyMzVjY2NhMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMGVlMTUwNDhhMzI0YTQ1NTAwNzQ4MTk4NWY3NGZiOTg3NzRiZDQ3MjA1NmY5MTRiYjcyODA1NzYwOWMzODkwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.UWz4Svmegdrvau7Pp2poZTIC_IBavEnIbvOnqlrQbds">
> 5. Kan vi ha "preview" på norsk?
> 
> <img alt="image" width="614" height="159" src="https://private-user-images.githubusercontent.com/155720728/483515874-c2638b2b-ef96-4a09-a5d5-58181ddaf9f3.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzcsIm5iZiI6MTc1NjQ2MzIzNywicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE1ODc0LWMyNjM4YjJiLWVmOTYtNGEwOS1hNWQ1LTU4MTgxZGRhZjlmMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNDEwMGZhY2JjYWExNGM2OGUwNDUyOGMwNWYwNDFkMDk0ZDY0YmJhMmI4OWEzYjRhNDM2Yzg1YjUzMDliZWViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.b7iVo7X_CaTKGSnzlgWd3BHjGMVGcpQfgMA1HOI-D94"> 😄
> 6. Flere av accountselectoreksempelet er brukket :(
> 
> <img alt="image" width="1229" height="588" src="https://private-user-images.githubusercontent.com/155720728/483516077-77ed7712-e768-4d20-b6cd-535440196b90.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzgsIm5iZiI6MTc1NjQ2MzIzOCwicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE2MDc3LTc3ZWQ3NzEyLWU3NjgtNGQyMC1iNmNkLTUzNTQ0MDE5NmI5MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kODIxNTBhYmM3YzFjOGRjN2Y0Mjk4NmI0Y2RhOGRmNDVlMjgyMGZkZjFiOTYzNjhjY2VlZmFhYTQ2ZWMxMTIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Xjgi--58BSOMq4sdNTwtC7OiUtMk9zhi1Cos2SczHrs"> Og ett av de ser ikke så bra ut.. <img alt="image" width="1180" height="592" src="https://private-user-images.githubusercontent.com/155720728/483516776-b1602646-aaac-4543-a291-bf1f7be9ddfd.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzgsIm5iZiI6MTc1NjQ2MzIzOCwicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE2Nzc2LWIxNjAyNjQ2LWFhYWMtNDU0My1hMjkxLWJmMWY3YmU5ZGRmZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iM2E1M2VjNGY1OTUzNDg5ZjU4NTZmYWEzNTE0NTRkN2EwZDFmMTZiODE3NWE2OGY3ZjUzOWNkNTQwNDNkMGEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0zkAYaIE1RxWp5MfA9atmKgodcv3NTI_YjVeXEDTIQ">
> 7. F.eks Cards: Trenger kanskje ikke dropdownen når det bare er ett eksempel? 🤔
> 
> <img alt="image" width="354" height="127" src="https://private-user-images.githubusercontent.com/155720728/483517488-34c5d40c-af07-4632-be14-3f442840db92.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0NjM1MzgsIm5iZiI6MTc1NjQ2MzIzOCwicGF0aCI6Ii8xNTU3MjA3MjgvNDgzNTE3NDg4LTM0YzVkNDBjLWFmMDctNDYzMi1iZTE0LTNmNDQyODQwZGI5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyOVQxMDI3MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YzM5MTdkYTlmZjcyMzA3MDc4OTVkMjNiNWNhN2E4NTc3Nzk4N2ZiN2M3ZDk4ZTY2NGM1M2QxNWUxMWM1ZWJmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Z-A-VOofBmGonWMLvTcp18ZE8473udZa7UGY6oPyJVw">
> Ser ut som det fortsatt er mange av eksemplene som bruker ikoner vi ikke støtter ❤️ Men vet ikke om du var ferdig da!

1. Litt usikker på hvordan jeg kan gjøre at den ikke kommer opp der, vet ikke om det er så stort problem?
2. Lagringen handler om at det du skriver inn blir lagret, selv ved refresh av siden. Teksten om ulagrede endringer ved tilbakestill er litt uklar, jeg skal forbedre den! =) 
3. Enig, jeg har gjort så det bare er ett eksempel nå! 
4. Kan gjøres i ny PR.
5. Endret til norsk
6. Forenklet og fjerner emojier!
7. Veldig godt forslag, fjernet dropdownen nå ved bare et eksempel
8. Fjernet alle Emojier nå! (Håper jeg...)

tuva-odegard commented:
Flott med endringer! 😄 

Har klikket meg gjennom enda mer og funnet enda litt til, sorry! 

1. Chips-eksempelet har feil 
<img width="1395" height="620" alt="image" src="https://github.com/user-attachments/assets/da9ce966-8721-4d94-84cd-7a15dab9ea35" />
2. Accountselector også :/ 
<img width="1381" height="611" alt="image" src="https://github.com/user-attachments/assets/94ea5e9f-f6f3-4ad5-a409-650c74229503" />

3. Ang. nr 2 på forrige kommentar, auto-lagringen ved refresh av siden ser ut til å fungere fint! 
Men hvis jeg gjør en endring og så prøver å bytte eksempel (med dropdown-lista) så får jeg opp denne meldingen 
<img width="965" height="542" alt="image" src="https://github.com/user-attachments/assets/30010c8d-c319-4d48-8cd9-86b775cd0d8b" />
Hvis jeg refresher siden og så bytter, går det fint. Så auto-lagringa fungerer fortsatt ikke helt som forventet 🤔 
3.1 I den samme ruta, er det noen grunn til at knappene er små? 😆 Synes godt de kan være vanlig størrelse, er jo plass til det.. 

4. Dette blir litt synsing fra min side (sammen med UX-prinispper), men kunne du flyttet auto-save ned til ved siden av Preview? 💯  Så er den litt nærmere det den hører til. Og helst den også på norsk 😄 
<img width="677" height="340" alt="image" src="https://github.com/user-attachments/assets/8c352ac7-2e1c-4892-9358-1fb1719f75bd" />

5. Jeg synes fortsatt også at dropdown skal få en label! ❤️  
<img width="280" height="129" alt="image" src="https://github.com/user-attachments/assets/8ff940b6-836f-48af-a65f-59e82334c6a9" />

tuva-odegard commented on .storybook/stories/LiveCodeEditor.stories.tsx:
```diff
@@ -0,0 +1,98 @@
+import type { Meta } from '@storybook/react';
+
+const meta: Meta = {
```
Hvor dukker denne opp? finner den ikke 🤔

hagenek replied:
Jeg sletter denne helt!

tuva-odegard commented on .storybook/shared/LiveCodeEditor.tsx:
```diff
@@ -0,0 +1,896 @@
+                                        ● Lagret
+                                    </span>
+                                )}
+                                <Button
```
Hvorfor ikke bare bruke ffe her? (tenker den fint kan være Tertiary forresten)

tuva-odegard commented on .storybook/shared/LiveCodeEditor.tsx:
```diff
@@ -0,0 +1,896 @@
+    );
+    const [hasUnsavedChanges, setHasUnsavedChanges] = useState<boolean>(false);
+    const [autoSave, setAutoSave] = useState<boolean>(true);
+    const [theme, setTheme] = useState<'light' | 'dark'>('light');
```
Ser ikke ut som at setTheme er brukt noe sted, og vi støtter vel heller ikke dark mode i storybook.. Tenker du gjerne kan fjerne all logikken her, det er mye kode som baserer seg på logikk vi ikke støtter 🤔

tuva-odegard commented on .storybook/shared/LiveCodeEditor.tsx:
```diff
@@ -0,0 +1,896 @@
+    return (
+        <div style={{ fontFamily: 'system-ui, -apple-system, sans-serif' }}>
+            <h3 className="ffe-h3">
+                {title} - {theme === 'light' ? 'Light' : 'Dark'} mode
```
Sånn som her 😊

tuva-odegard commented on .storybook/shared/LiveCodeEditor.tsx:
```diff
@@ -0,0 +1,896 @@
+            {/* Custom confirmation dialog */}
+            {showConfirmDialog && (
+                <div
+                    style={{
```
Ikke viktig, men hvorfor ikke bruke ffe-modal her? 🤔

hagenek replied:
Kan ta det i en ny PR =)

tuva-odegard commented on .storybook/shared/LiveCodeEditor.tsx:
```diff
@@ -0,0 +1,896 @@
+                        `
+**🔥 Live ${storageKey} Code Editor**
+
+Skriv JSX-kode og se resultatet umiddelbart! Editoren støtter:
```
Hvor dukker det her opp? Er det bare når man skal bruke den?

hagenek replied:
Dette er hvis man ikke gir noen description når man lager en live code editor spesifikt for en komponent. Har kortet ned denne alternative beskrivelsen betraktelig nå.

tuva-odegard commented on packages/ffe-accordion-react/src/LiveCodeEditor.stories.tsx:
```diff
@@ -0,0 +1,50 @@
+    defaultTemplate: 'basic',
+    title: 'Accordion Live Code Editor',
+    description:
+        'Live code editor for Accordion komponenten med alle templates og funksjonalitet.',
```
"med alle templates og funksjonalitet." er litt intentsigende 😅 
Kanskje det kan stå noe som at "Her kan du endre på et eksempel med bruk av Accordion" eller noe?

tuva-odegard commented on packages/ffe-account-selector-react/src/LiveCodeEditor.stories.tsx:
```diff
@@ -0,0 +1,102 @@
+        docs: {
+            description: {
+                component:
+                    'Comprehensive live code editor for all account selector components in the FFE AccountSelector package. Explore AccountSelector and AccountSelectorMulti with interactive examples for single and multi-account selection scenarios.',
```
Dette burde være på norsk. Gjelder alle live kode editorene! Og burde kanskje ikke si at det er flere eksempler når det bare er ett nå 🤔

tuva-odegard commented on packages/ffe-account-selector-react/src/LiveCodeEditor.stories.tsx:
```diff
@@ -0,0 +1,102 @@
+    defaultTemplate: 'simple',
+    title: 'AccountSelector Live Code Editor',
+    description:
+        'Comprehensive live code editor for all account selector components in the FFE AccountSelector package. Explore AccountSelector and AccountSelectorMulti with interactive examples for single and multi-account selection scenarios.',
```
Norsk? 🫶

tuva-odegard commented on .storybook/shared/LiveCodeEditor.tsx:
```diff
@@ -0,0 +1,896 @@
+    defaultTemplate = Object.keys(templates)[0],
+    storageKey,
+    title = 'Live Code Editor',
+    description = 'Skriv JSX-kode og se resultatet umiddelbart!',
```
haha, det blir så tydelig at det ikke er vi som har skrevet det 🙃 
Hva med "Her kan du redigere komponenten og se resultatet før du tar den i bruk." eller noe?

hagenek committed:
feat(.storybook): forbedret featuren etter kommentarer i PR
